lib/ReactTags.js

```diff
@@ -84,6 +88,7 @@ class ReactTags extends Component {
     this.resetAndFocusInput = this.resetAndFocusInput.bind(this);
     this.handleSuggestionHover = this.handleSuggestionHover.bind(this);
     this.handleSuggestionClick = this.handleSuggestionClick.bind(this);
+    this.filteredSuggestions = this.filteredSuggestions.bind(this);
```

Comment:
why binding here ?

Reply:
To prevent issues when calling this.props

Reply:
same as above arrow functions